scss/_inheritance.scss

```diff
@@ -0,0 +1,20 @@
+%centering {
+  display: flex;
```

Comment:
You can use mixin for flex containers which would contain display: flex, justify-content, align-items, flex-wrap properties for example.

Reply:
Done